src/App.tsx

```diff
@@ -1,16 +1,27 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [
+    moviesFromServerWithNew,
+    setMoviesFromServerWithNew,
+  ] = useState(moviesFromServer);
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesFromServerWithNew} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie onAdd={(movie) => {
```

Comment:
```suggestion
        <NewMovie onAdd={handleAdding}
```
Consider moving this logic from jsx